config/routes.rb

```diff
@@ -48,6 +49,9 @@
     resources :observations, only: [:create, :destroy]
   end
 
+  resources :reports, only: [:index, :show, :create, :destroy] do
```

Comment:
think we don't need this `do` and `end` unless there's a nested route we forgot here?